packages/bpk-component-scrollable-calendar/src/BpkScrollableCalendarGrid-test.js

```diff
@@ -0,0 +1,152 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { mount } from 'enzyme';
+import isWeekend from 'date-fns/is_weekend';
+
+import { colorRed500 } from 'bpk-tokens/tokens/base.es6';
+import BpkCalendarScrollGrid from './BpkScrollableCalendarGrid';
+import BpkCalendarScrollDate from './BpkScrollableCalendarDate';
+
+import { weekDays, formatDateFull, formatMonth } from '../test-utils';
+
+describe('BpkCalendarScrollGrid', () => {
+  it('should render correctly with "showWeekendSeparator" attribute set to false', () => {
```

Comment:
Probably good to have a test case for when all the required props are set but none of the optional ones 👍